docs/sources/copyrights.rst

```diff
@@ -1,47 +1,81 @@
 
-COPYRIGHT.   The program, content and related documentation are copyrighted.  You may not copy the program, its content or the documentation (or this license) except to create one backup copy as set forth above and to load the program into the computer as part of executing the program.  Among other things, you may not copy the program onto any shared repository or similar system.  Any and all other copies of the program, its content or its documentation made by you are in violation of this license. 
+2.2 Customer represents, covenants, and warrants that Customer will use the Software only in compliance with 3Laws’ standard published policies and codes of conduct then in effect and all applicable laws and regulations.  Customer hereby agrees to indemnify and hold harmless 3Laws against any damages, losses, liabilities, settlements and expenses (including without limitation costs and attorneys’ fees) in connection with any claim or action that arises from an alleged violation of any of the foregoing or otherwise from Customer’s use of Software. Although 3Laws has no obligation to monitor Customer’s use of the Software, 3Laws may do so and may prohibit any use of the Software (or disable content or data) it believes may be (or alleged to be) in violation of the foregoing or any other term of this Agreement.
 
-OWNERSHIP.    You agree that you neither own nor hereby acquire any claim or right of ownership to the program, content and documentation or to any related patents, copyrights, trademarks or other intellectual property.  3Laws Robotics, Inc. retains ownership of the documentation and all copies thereof and the program and its content and all subsequent copies of the program and content at all times, regardless of the form or media in or on which the original or other copies may subsequently exist.  This license is not a sale of the original or any subsequent copy.
+2.3 Customer acknowledges that the Software (i) records and logs certain usage information related to the Software and related robotic systems, and (ii) requires license keys from 3Laws in order to function.  Customer shall not disable or tamper with such recording, logging and/or license key functionality in any way and shall provide 3Laws with reasonable access (either direct or remote, at 3Laws’ option) to all systems on which the Software is installed, operated or used for purposes of monitoring or retrieving any such information or license keys and otherwise cooperate with 3Laws’ efforts to obtain and monitor such information and 3Laws license keys. 
```

Comment:
Is it standard ? Seems that we ask for full access to their system

Reply:
I brought this issue up with Amir.